spec/spec_helper.rb

```diff
@@ -17,6 +17,9 @@
 #
 # See http://rubydoc.info/gems/rspec-core/RSpec/Core/Configuration
 RSpec.configure do |config|
+  # remove unnecessary stack trace messages from files that aren't ours
+  config.backtrace_exclusion_patterns << /\.rvm\/gems/
```

Comment:
Nice find Orion!  I like this a lot!  Eliminates all the rspec error messages that are less relevant